django/contrib/auth/tests/custom_user.py

```diff
@@ -221,6 +224,31 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
     class Meta:
         app_label = 'auth'
 
+
+class CustomUserUUIDasPK(AbstractBaseUser, PermissionsMixin):
```

Comment:
One more thing - why are you declared model here and importing it in another file?